### ISSUE TITLE
Can not add more welders to Icarus station

### ISSUE THREAD

**Issue by Thebadopinion** [DraconisCluster/DX-Issues#55]
** tl; dr **
--type of issue (bug, fr, etc), short what and why--no one can add more welders to wariorx station icarus. Also no matter how many welders wariorx owns on the station it will always say I own six in my limits, even if I own zero.

** Full info **
--long what and why--due to the welders eating so much of our comps two days ago we grind the welders off of the station and converted them back to regular MCRN and UNN welders when we went to put the welders on the station it would not allow me and three others to do so it would constantly say max limit reached we fix ship. We have grind down those welders and replace them with other welders. We’ve hanger it and we’ve waited for server restarts. My limit is always on the station regardless of how many welders is actually on the station, the same has applied to Ollie  and Nemesis 

**Screenshots**
--If applicable, add screenshots to help explain your problem.--

**Comment by Terif1** [comment]
When the limits seem not to be updating, have you tried running !blocklimit update mylimit ?

**Comment by draconb** [comment]
What does it say in !blocklimit mylimits

**Comment by draconb** [comment]
Station welders are separate category from other welders, other welders are limited to 6 per grid, 12 total per person, station welders are limited to 6 per person
